qutebrowser/config/configdata.yml

```diff
@@ -958,13 +958,14 @@ downloads.location.suggestion:
 completion.open_categories:
   type:
     name: FlagList
-    valid_values: [searchengines, quickmarks, bookmarks, history]
+    valid_values: [searchengines, quickmarks, bookmarks, history, buffers]
     none_ok: true
   default:
     - searchengines
     - quickmarks
     - bookmarks
     - history
+    - buffers
```

Comment:
FWIW, I don't think that buffers should be here by default, and if it should, it should be at the top.